src/data/things/album.js

```diff
@@ -357,6 +358,11 @@ export class Album extends Thing {
 
       include: album =>
         album.hasCoverArt,
+
+      getMatchableNames: album =>
+      (album.alwaysReferenceByDirectory 
+        ? [] 
+        : [album.name]),
```

Comment:
Indent the body of an arrow function. The ternary is good, just indent these three lines by two spaces:

```suggestion
      getMatchableNames: album =>
        (album.alwaysReferenceByDirectory 
          ? [] 
          : [album.name]),
```